.evergreen/config.yml

```diff
@@ -2443,12 +2428,12 @@ buildvariants:
   rules:
     # Server versions 3.6+ support snappy and zlib.
     - if:
+        platform: "*"
         python-version: "*"
         c-extensions: "*"
         compression: ["snappy", "zlib"]
       then:
         add_tasks:
-          - "test-5.0-standalone"
```

Comment:
Why get rid of 5.0 testing?

Reply:
We still test against 5.0 the "test-5.0-standalone" here was redundant.

Reply:
Ah, got it.